src/desktop/__tests__/ui/global/__snapshots__/about.test.jsx.snap

```diff
@@ -18,14 +18,34 @@ exports[`About component Render the component 1`] = `
     </h1>
     <h2>
       v
-      0.5.1
+      0.5.2
        
       <small>
         BETA
       </small>
     </h2>
     <article>
       <Scrollbar>
+        <h5>
+          0.5.2
```

Comment:
We might want to think about how we can make this test less flaky -- it will fail and require a snapshot update each time we edit the changelog

Reply:
We should move the change log to `CHANGELOG.txt` and load the content from there. Markdown is already supported, so no big changes required. I'll do a PR.